### ISSUE TITLE
Service center search button doesn't have accessible name or function

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/wp-proudcity#1896]
> The search button under "Your Local Services" in My Services tab panel does not have an accessible name (i.e., it's a blank button for screen reader users). It also doesn't seem to function for any type of user. 
`<button class="btn btn-lg btn-default hidden-xs"><i aria-hidden="true" class="fa fa-search"></i></button>`
> Recommend adding a hidden name of "Search" and making it a fully functional button like the site search button.

**Comment by jlyon** [comment]
This "button" serves no functional purpose. That text field is a Google address typeahead field that is automatically submitted when you select an address.  There really isn't a way for us to make that button fuctional. Suggestion: Just make the "search" icon a part of the field like this and remove the button:
<img width="410" alt="Screen Shot 2021-11-04 at 10 43 41 PM" src="https://user-images.githubusercontent.com/550909/140464135-20bf2383-4871-4c06-9a68-d543f8ec2d34.png">

**Comment by kevindherman** [comment]
@lukefretwell what do you think of @jlyon's suggestion here? If that works, lets move forward with it.

**Comment by lukefretwell** [comment]
@jlyon @kevindherman yep

**Comment by jlyon** [comment]
This change has been made and deployed platform-wide @kevindherman

**Comment by kevindherman** [comment]
@jlyon thanks. 

Should this be moved to ready or just closed?
